templates/admin/jqadm/product/item-media.php

```diff
@@ -73,9 +73,21 @@ class="btn btn-card-header act-delete fa" tabindex="<?= $this->get( 'tabindex' )
 								<input class="item-url" type="hidden"
 									v-bind:name="`<?= $enc->js( $this->formparam( ['media', '_idx_', 'media.url'] ) ) ?>`.replace('_idx_', idx)"
 									v-model="item['media.url']">
-								<img v-if="item['media.preview']" class="item-preview"
+								<img v-if="item['media.preview'] && (item['media.mimetype'].startsWith('image') || item['media.mimetype'].startsWith('video'))" class="item-preview"
```

Comment:
I think this should be only `item['media.mimetype'].startsWith('audio')` so mime icons for PDF, etc. are still displayed.

Reply:
Why audio? Here it is all about EXCLUDING audio...